apps/meteor/app/lib/server/functions/updateMessage.ts

```diff
@@ -99,6 +100,8 @@ export const updateMessage = async function (
 				id: msg._id,
 				data: msg,
 			});
+
+			void notifyListenerOnRoomChanges(message.rid);
```

Comment:
the only scenario I can think this would be needed is the case when we update the last message of a room.. if this is really the case, should we be smart and only broadcast if that is really the case? just thinking out loud here ;)